src/libmeasurement_kit/nettests/runnable.cpp

```diff
@@ -238,74 +238,131 @@ std::string Runnable::generate_output_filepath() {
     return filename.str();
 }
 
+void Runnable::contact_bouncer(Callback<Error> cb) {
+    if (!needs_bouncer) {
+        cb(NoError());
+        return;
+    }
+    auto bouncer = options.get(
+        "bouncer_base_url", std::string{"https://a.collector.ooni.io/bouncer"});
```

Comment:
True. Like it is done for the collector, we can use the testing server. It is `bouncer.test.ooni.io` on port 443 using https.